allennlp/modules/stacked_elmo_lstm.py

```diff
@@ -0,0 +1,146 @@
+        `A Theoretically Grounded Application of Dropout in Recurrent Neural Networks
+        <https://arxiv.org/abs/1512.05287>`_ .
+    state_projection_clip_value: ``float``, optional, (default = 3)
+        The magnitude with which to clip the hidden_state after projecting it.
+    memory_cell_clip_value: ``float``, optional, (default = 3)
+        The magnitude with which to clip the memory cell.
+    """
+    def __init__(self,
+                 input_size: int,
+                 hidden_size: int,
+                 cell_size: int,
+                 num_layers: int,
+                 recurrent_dropout_probability: float = 0.0,
+                 memory_cell_clip_value: float = 3.0,
+                 state_projection_clip_value: float = 3.0) -> None:
+        super(ElmoLstm, self).__init__()
+
+        # Required to be wrapped with a :class:`PytorchSeq2SeqWrapper`.
+        self.input_size = input_size
+        self.hidden_size = hidden_size
+        self.num_layers = num_layers
+
+        forward_layers = []
+        backward_layers = []
+
+        lstm_input_size = input_size
+        go_forward = True
+        for layer_index in range(num_layers):
+            forward_layer = ElmoLstmCell(lstm_input_size,
+                                         hidden_size,
+                                         cell_size,
+                                         go_forward,
+                                         recurrent_dropout_probability,
+                                         memory_cell_clip_value,
+                                         state_projection_clip_value)
+            backward_layer = ElmoLstmCell(lstm_input_size,
+                                          hidden_size,
+                                          cell_size,
+                                          not go_forward,
+                                          recurrent_dropout_probability,
+                                          memory_cell_clip_value,
+                                          state_projection_clip_value)
+            lstm_input_size = hidden_size
+
+            self.add_module('forward_layer_{}'.format(layer_index), forward_layer)
+            self.add_module('backward_layer_{}'.format(layer_index), backward_layer)
+            forward_layers.append(forward_layer)
+            backward_layers.append(backward_layer)
+        self.forward_layers = forward_layers
+        self.backward_layers = backward_layers
+
+    def forward(self,  # pylint: disable=arguments-differ
+                inputs: PackedSequence,
+                initial_state: Optional[Tuple[torch.Tensor, torch.Tensor]] = None):
+        """
+        Parameters
+        ----------
+        inputs : ``PackedSequence``, required.
+            A batch first ``PackedSequence`` to run the stacked LSTM over.
+        initial_state : Tuple[torch.Tensor, torch.Tensor], optional, (default = None)
+            A tuple (state, memory) representing the initial hidden state and memory
```

Comment:
This docstring is inconsistent with the one in the class definition above. I'd just eliminate it otherwise it's repetitive and hard to maintain (need to change things in multiple places for a single change).